<!DOCTYPE html>
<html><head><title>InvokeStartPublishingAsync</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">4 references to InvokeStartPublishingAsync</div><div class="rA">Azure.Messaging.EventHubs.Tests (4)</div><div class="rG" id="Azure.Messaging.EventHubs.Tests"><div class="rF"><div class="rN">Producer\EventHubBufferedProducerClientTests.cs (4)</div>
<a href="../Producer/EventHubBufferedProducerClientTests.cs.html#885"><b>885</b>Assert.That(async () =&gt; await <i>InvokeStartPublishingAsync</i>(mockBufferedProducer.Object, cancellationSource.Token), Throws.InstanceOf&lt;TaskCanceledException&gt;());</a>
<a href="../Producer/EventHubBufferedProducerClientTests.cs.html#905"><b>905</b>await <i>InvokeStartPublishingAsync</i>(mockBufferedProducer.Object, cancellationSource.Token);</a>
<a href="../Producer/EventHubBufferedProducerClientTests.cs.html#944"><b>944</b>Assert.That(async () =&gt; await <i>InvokeStartPublishingAsync</i>(mockBufferedProducer.Object, cancellationSource.Token), Throws.TypeOf(expectedException.GetType()), &quot;The attempt to start publishing should have surfaced an exception.&quot;);</a>
<a href="../Producer/EventHubBufferedProducerClientTests.cs.html#990"><b>990</b>await <i>InvokeStartPublishingAsync</i>(mockBufferedProducer.Object, cancellationSource.Token);</a>
</div>
</div>
</body></html>